APP/ev_chargers/lib/screens/navbar.dart

```diff
@@ -0,0 +1,125 @@
+import 'package:ev_chargers/style.dart';
+import 'package:flutter/material.dart';
+
+class Navbar extends StatefulWidget {
+  Color backgroundColor;
+  Color elementColor;
+  List<String> elemTags;
+  List<Icon> icons;
+  int current = 0;
+  Function(int) onPressed;
+
+  Navbar(this.backgroundColor, this.elementColor, this.elemTags, this.icons, this.current,
+      this.onPressed);
+
+  @override
+  _NavbarState createState() =>
+      _NavbarState(this.backgroundColor, this.elementColor, this.elemTags, this.icons, this.onPressed);
+}
+
+class _NavbarState extends State<Navbar> {
+  Color backgroundColor;
+  Color elementColor;
+  List<String> elemTags;
+  List<Icon> icons;
+
+  Function(int) onPressed;
+  _NavbarState(this.backgroundColor, this.elementColor, this.elemTags, this.icons, this.onPressed);
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+        color: Colors.amber,
+        padding: EdgeInsets.only(bottom: MediaQuery.of(context).padding.bottom),
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+          children: _getChildren(),
+        ));
+  }
+
+  List<Widget> _getChildren() {
+    List<Widget> ret = [];
+    for (int i = 0; i < elemTags.length; i++) {
```

Comment:
You can try to play with `forEach`, or with [this](https://api.dart.dev/stable/2.2.0/dart-core/List/asMap.html).

Reply:
It could be done by I need iteration count as well, enumerate from python would be perfect but I don't know if something like that exists here and we are on tight schedule so I chose this